### DISCUSSION TITLE
How to set a page as homepage?

### DISCUSSION THREAD

**Discussion by ooker777** [ObsidianPublisher/obsidian-github-publisher#140] [Q&A — Template] [answered]
Basic question, but I fail to find the answer. I suppose one should have an index.html in the site folder, but how to tell mkdocs to do that? I want to set it in Obsidian, if possible. I also notice an overrides folder, but playing around doesn't change anything. I try to turn it off in the mkdocs.yml file but still no use

**Comment by Mara-Li** [comment]
index.md is the name for an index home page. It needs to be in the root.
You can check the repo of the docs if you want to see a structure : https://github.com/ObsidianPublisher/obsidian-mkdocs-publisher-docs

**Reply by ooker777** [reply]
so that means I must manually rename a file I want to be index.html, not assigning it via Obsidian?

**Comment by Mara-Li** [comment]
Again, it's a markdown file.

And no, you can use the frontmatter path to change the filepath in the repo

**Reply by ooker777** [reply, marked as answer]
oops. I didn't know that you need to set it `title: index`. I guess allowing the key `home` with value `true` would be more intuitive. I make a pr here https://github.com/ObsidianPublisher/obsidian-mkdocs-publisher-docs/pull/8
